### ISSUE TITLE
(Contains fix) Makefile doesn't point to the correct pytest binary

### ISSUE THREAD

**Issue by kjohnson0451** [ansible-community/ansible-bender#238]
I was following the instructions of the [Contributing page](https://ansible-community.github.io/ansible-bender/build/html/contributing.html). Upon the `make check` step, I run into this error

```
PYTHONPATH=/root/ansible-bender PYTHONDONTWRITEBYTECODE=yes pytest-3 --cov=ansible_bender -l -v ./tests/
/bin/sh: 1: pytest-3: not found
make: *** [Makefile:13: check] Error 127
```

Background info: My pytest version is 6.1.2. Python version 3.7.3. OS is Debian 10.6

FIX: In [Makefile:7](https://github.com/ansible-community/ansible-bender/blob/master/Makefile#L7) I changed `PYTEST_EXEC := pytest-3` to `PYTEST_EXEC := pytest`. This is just a hot-edit in my workspace for now.

Should this be pushed upstream?

**Comment by TomasTomecek** [comment]
Hi Keith, yes, we can change this in upstream - the reason the default is `pytest-3` is that's Fedora Linux had while moving from python 2 to python 3. I just checked that `python3-pytest` on Fedora now has `pytest` and `pytest-3` - please open a PR and we can merge that.

**Comment by TomasTomecek** [comment]
Fixed by #242
